tests/events/39-task-event-template-all/suite.rc

```diff
@@ -11,7 +11,10 @@
         graph = "foo"
 [runtime]
     [[foo]]
-        script = cylc message -p CUSTOM "cheesy peas"
+        script = """
+wait "${CYLC_TASK_MESSAGE_STARTED_PID}" 2>/dev/null || true
```

Comment:
NIce, I forgot about this... need to change my advice on use of `cylc checkpoint` in task scripting in #2469.

Reply:
(done)